src/components/PlayerList.jsx

```diff
@@ -0,0 +1,40 @@
+//takes in a prop that is a list of player or teams, and then maps through them and returns a link of each one.
+//an array of something as a prop
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getPlayers } from '../services/players';
+
+function PlayerList() {
+  const [players, setPlayers] = useState([]);
```

Comment:
all of this business logic from line 9 to line 20 should be in the Players.jsx file instead of this component, with the players array passed down as a prop